frontends/user-articles/UserArticlesSidebar.tsx

```diff
@@ -135,6 +135,9 @@ export const UserArticlesSidebar: FC = () => {
           </Link>
         )}
       </ArticleStatus>
+      <Link href={`${WEB_APP_ROOT_DIR}/docs/write-article_introduction_quick-start`}>
```

Comment:
<img width="1440" alt="Screenshot 2023-07-05 at 17 44 48" src="https://github.com/Juratbek/upper_frontend/assets/55176773/3f6bac49-6a26-4c0f-aee4-dedcb1646818">

- [ ] add a divider
- [ ] change the text
- [ ] "Foydalanish uchun qo'llanma" text will be a link